src/common/src/catalog/physical_table.rs

```diff
@@ -29,6 +29,8 @@ use crate::util::sort_util::ColumnOrder;
 pub struct TableDesc {
     /// Id of the table, to find in storage.
     pub table_id: TableId,
+
+    pub table_name: Option<String>,
```

Comment:
Hmmm, you are reusing `TableDesc` to describe an external table, right?

https://github.com/risingwavelabs/risingwave/blob/efa436c319ee9a9ce034453ef513e1b00ec231ce/src/frontend/src/handler/create_table.rs#L805

This looks a bit wierd to me. In my mind, the external CDC upstream table is not a table but completely different thing.

Note that

- Here `physical_table::TableDesc` (or `StorageTableDesc` in proto definition) is clearly for storage. 
- Another `TableCatalog` (or `catalog.Table` in proto definition) is for optimizer and executors

I think you need to introduce a new struct like `CdcTableDesc` to save these CDC-related things. Anyway, putting these into `physical_table::TableDesc` looks wierd.